packages/models-library/src/models_library/api_schemas_directorv2/errors.py

```diff
@@ -15,3 +16,11 @@ class ErrorType(BaseModel):
 
 class ErrorEnveloped(BaseModel):
     error: ErrorType
+
+
+class BaseRpcError(OsparcErrorMixin, Exception):
```

Comment:
please use `RPCInterfaceError`

see examples in https://github.com/itisfoundation/osparc-simcore/blob/020e2dfc353ebb4c94eceba3a91c762703b2ec0f/packages/service-library/tests/rabbitmq/test_rabbitmq_rpc_router.py

Reply:
THOUGHT: we need amore systematic way to create rpc apis
- server-side
- client-side
- shared schemas and exceptions
- server mocks


And then, **refactor all the rpc API** to follow those. Until then, lets start to make sure in the reviews that we are as consistent as possible